.github/workflows/tests.yaml

```diff
@@ -475,6 +475,16 @@ jobs:
                       build \
                       --copy-artifacts-to objdir-clone \
                    "
+            - name: Generate an argument environment file
+              run: |
+                  touch /tmp/test_env.yaml
```

Comment:
```suggestion
                  rm -f /tmp/test_env.yaml
                  touch /tmp/test_env.yaml
```

Reply:
Thinking we should ensure file exists and is clear. Either rm -f first or echo empty to clear it.

Reply:
Added and Fixed.